packages/fxa-payments-server/src/routes/Checkout/index.tsx

```diff
@@ -135,6 +135,67 @@ export const Checkout = ({
     console.log('not yet implemented: refreshSubscriptions');
   }
 
+  const fxaLegalBlurb = (
+    <Localized
+      id="accounts-legal-copy"
+      elems={{
+        fxaPrivacyLink: (
+          <a
+            href={`${config.servers.content.url}/legal/privacy`}
+            target="_blank"
+            rel="noopener noreferrer"
+          ></a>
+        ),
+        fxaTermsLink: (
+          <a
+            href={`${config.servers.content.url}/legal/terms`}
+            target="_blank"
+            rel="noopener noreferrer"
+          ></a>
+        ),
+      }}
+    >
+      <p className="legal-blurb-addendum">{`<fxaTermsLink>Terms of Service</fxaTermsLink> | <fxaPrivacyLink>Privacy Notice</fxaPrivacyLink>`}</p>
+    </Localized>
+  );
+
+  const productMetadata = selectedPlan?.product_metadata || {};
```

Comment:
We have some existing helpers here so that plan metadata can override product metadata if desired, and so that we use localized metadata strings if provided: `productDetailsFromPlan`. This was previously used to obtain these URLs in `PaymentForm`. Please use that helper here.